src/contract/client.ts

```diff
@@ -26,34 +26,40 @@ export class Client {
   ) {
     this.spec.funcs().forEach((xdrFn) => {
```

Comment:
I'm confused: why is there a reference to `this.spec` here when `spec` is also passed as a parameter and is no longer being used?

Reply:
Lolll you're so right - I remember now. Thank you for explaining :pray: I clearly don't leverage the fancy features of TS often enough.